pkgs/racket-test-core/tests/racket/fasl.rktl

```diff
@@ -72,6 +72,9 @@
 (test #f eq? #px"hello" (fasl->s-exp (s-exp->fasl #px"hello") #:datum-intern? #f))
 (test #f eq? #rx#"hello" (fasl->s-exp (s-exp->fasl #rx#"hello") #:datum-intern? #f))
 (test #f eq? #px#"hello" (fasl->s-exp (s-exp->fasl #px#"hello") #:datum-intern? #f))
+(test #f eq? #rx"[/\u5C][. ]+ap" (fasl->s-exp (s-exp->fasl #rx"[/\u5C][. ]+ap") #:datum-intern? #f))
```

Comment:
Can you change these to use `equal?` and check that they produce the same thing? Also avoid duplicating the regexp.